index.html

```diff
@@ -24,9 +24,32 @@
         <li><a href="#">Contact</a></li>
       </ul>
     </div>
-    <nav id="navbar">
+    <button type="button" id="navbar">
       <i class="fa fa-bars"></i>
-    </nav>
+    </button>
+    <nav class="header-nav-mobile hidden">
+      <button type="button" class="btn-close-nav">
+        <img src="./images/icons8-cancel-32.png" alt="cancel icon">
+      </button>
+      <ul>
+          <li>
+              <a href="#" class="link-portfolio">
+                  Portfolio
+              </a>
+          </li>
+          <li>
+              <a href="#" class="link-about">
+                  About
+              </a>
+          </li>
+          <li>
+              <a href="#" class="link-contact">
+                  Contact
+              </a>
```

Comment:
- [ ] Kindly make sure to display the correct part of the page when the menu item is clicked as per the loom video attached.